src/org/exist/xquery/lib/kwic.xql

```diff
@@ -142,16 +172,16 @@ declare function kwic:get-summary($root as node(), $node as element(),
 	@param $callback (optional) reference to a callback function which will be called
 	once for every text node before it is appended to the displayed text. The function
 	should accept 2 parameters: 1) a single text node, 2) a string indicating the
-	current direction in which text is appended, i.e. "before" or "after". The function
-	may return the empty sequence if the current node should be ignore (e.g. if it belongs
-	to a "footnote" which should not be displayed). Otherwise it should return a single
-	string.
+	current direction in which text is appended, i.e. $kwic:MODE_BEFORE or $kwic:MODE_AFTER.
```

Comment:
@joewiz Yes, because the variables are public so the developer can use them from their own modules without caring what the values are.